snuba/cli/devserver.py

```diff
@@ -170,6 +170,30 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                     "--auto-offset-reset=latest",
                 ],
             ),
+            (
```

Comment:
there is an else-branch where you should add the same consumer again. normally in devserver, SEPARATE_SCHEDULER_EXECUTOR_SUBSCRIPTIONS_DEV is False, and all subscription executors are running in a single consumer (for lower resource usage)

Reply:
added https://github.com/getsentry/snuba/pull/6396/commits/d8035005090d523bc427f6d871042d6e5189b575